### PR TITLE
Add anti-debugging, obfuscation

### PR DESCRIPTION
**Pull request by sourque**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -118,20 +118,27 @@ points = 20
...
     [[check.pass]] # You can code multiple pass conditions
-    type="Command" # If any pass, the check passes
+    type="Command" # they must ALL succeed for the check to pass
     arg1="ufw status"
 
 [[check]]
-# If no points are specified, they are auto-calculated
-# out of 100 points (ex. 50 specified points, 5 checks
-# with no points specified-- they're 10 points each)
+# If no points are specified, they are auto-calculated out of 100
```
Comment (safinsingh) [inline README.md]: does this imply fractional points?

File: src/scoring.go [hunk 2]
```diff
@@ -102,19 +106,41 @@ func scoreChecks() {
...
+
+		// For multiple pass conditions, will only be true if ALL of them are
+		for _, result := range passStatus {
+			status = status && result
+			if !status {
+				break
+			}
+		}
+		if debugEnabled {
+			infoPrint(fmt.Sprint("Result of all pass check was ", status))
```
Comment (safinsingh) [inline src/scoring.go:126]: `infoPrint("Result of all pass check was " + status)`

File: src/scoring.go [hunk 3]
```diff
@@ -102,19 +106,41 @@ func scoreChecks() {
...
+				status = true
 				break
 			}
 		}
 		for _, condition := range check.Fail {
 			failStatus := processCheckWrapper(&check, condition.Type, condition.Arg1, condition.Arg2, condition.Arg3)
-			if reverseEnabled {
-				failStatus = !failStatus
+			if debugEnabled {
+				infoPrint(fmt.Sprint("Result of fail check was ", failStatus))
```
Comment (safinsingh) [inline src/scoring.go:143]: infoPrint("Result of fail check was " + failStatus)

File: src/utility_windows.go [hunk 4]
```diff
@@ -74,6 +88,10 @@ func sendNotification(messageString string) {
 // speed things up, as well as some other flags) to run commands on the host
 // system and retrieve the return value.
 func rawCmd(commandGiven string) *exec.Cmd {
+	if debugEnabled {
+		cmdInput := fmt.Sprintln("powershell.exe", "-NonInteractive", "-NoProfile", "Invoke-Command", "-ScriptBlock", "{ "+commandGiven+" }")
```
Comment (safinsingh) [inline src/utility_windows.go]: `cmdInput := "powershell.exe -NonInteractive -NoProfile Invoke-Command -ScriptBlock { "+commandGiven+" }"`

**Review by safinsingh — CHANGES_REQUESTED** [review]
See below, everything else lgtm

File: src/scoring.go (hunk 2 above)
Comment (sourque) [reply src/scoring.go:126]: I don't think you can concat strings and bools like that. It is ugly with `Sprint` but I can't think of any cleaner solutions.

**Review by safinsingh — APPROVED** [review]
Spicy

**Pull request merged**